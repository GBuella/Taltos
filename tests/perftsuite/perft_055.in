setboard R6r/8/8/2K5/5k2/8/8/r6R w - - 0 1
echo R6r/8/8/2K5/5k2/8/8/r6R w - - 0 1
qperft 1
qperft 2
qperft 3
qperft 4
qperft 5
qperft 6
