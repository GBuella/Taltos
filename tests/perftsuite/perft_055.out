R6r/8/8/2K5/5k2/8/8/r6R w - - 0 1
36
1027
29215
771461
20506480
525169084
